src/Avalonia.Controls/Flyouts/PopupFlyoutBase.cs

```diff
@@ -119,6 +126,16 @@ public FlyoutShowMode ShowMode
             set => SetValue(OverlayInputPassThroughElementProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets a value describing how the popup position will be adjusted if the
+        /// unadjusted position would result in the popup being partly constrained.
+        /// </summary>
+        public PopupPositionerConstraintAdjustment PlacementConstraintAdjustment
```

Comment:
I would use `<inheritdoc cref="Popup.PlacementConstraintAdjustment"/>` so we don't duplicate this. Note that all other Popup inherited properties do this and are in the section above (sometimes it's good to keep ordering but that a nit).

Reply:
Didn't know that was a thing. Will update it next week, hopefully Monday latest. Thx.